ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs

```diff
@@ -387,15 +382,22 @@ private void StageDamageSelectionChanged()
         {
             if (SelectedStageDamageElement != null && SelectedStageDamageElement.ChildElement != null)
             {
-                AggregatedStageDamageElement elem = (AggregatedStageDamageElement)SelectedStageDamageElement.ChildElement;
-                List<StageDamageCurve> stageDamageCurves = elem.Curves.Where(curve => curve.ImpArea.ID == CurrentImpactArea.ID).ToList();
+                List<StageDamageCurve> stageDamageCurves = GetStageDamageCurves();
 
                 DamageCategories.Clear();
                 DamageCategories.AddRange(stageDamageCurves);
                 SelectedDamageCurve = DamageCategories[0];
             }
         }
 
+        //todo: check that these selected items aren't null?
+        private List<StageDamageCurve> GetStageDamageCurves()
+        {
+            AggregatedStageDamageElement elem = (AggregatedStageDamageElement)SelectedStageDamageElement.ChildElement;
+            List<StageDamageCurve> stageDamageCurves = elem.Curves.Where(curve => curve.ImpArea.ID == CurrentImpactArea.ID).ToList();
```

Comment:
You don't need to keep around an output variable, just return elem.Curves.Where(...)